### PR TITLE
Add a benchmark to verify performance

### LINKED ISSUE #54: Byte Array raise a problem during consuming
**Describe the bug**
Using `byte[]` creates problem during data extraction.

**To Reproduce**
1. Use a consumer like `KafkaConsumer<int, byte[]>`
2. Try to consume using `Poll`
3. When data are received try to get the `Value` of the `ConsumerRecord<int, byte[]>`
4. An exception is raised

**Expected behavior**
The `byte[]` received shall be available to the API user.

**Desktop (please complete the following information):**
 - OS: All
 - Version: latest
 
**Additional context**
The exceptions seems be raised from JCOBridge: check if the call wasn't done in the correct way.

### PR DESCRIPTION
**Pull request by masesdevelopers**
## Description
This PR introduce new tests (benchmarks) to verify the capability of KNet. Meanwhile some new classes were introduced to optimize produce/consume operations and some bug fix were done.

## Related Issue
closed #54
#53

## Motivation and Context
Requested from issue

## How Has This Been Tested?
Tested locally using various Apache Kafka clusters

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/java/knet/src/main/java/org/mases/knet/clients/consumer/KNetConsumer.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.annotation.InterfaceStability;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Properties;
+
+@InterfaceStability.Evolving
```
Comment (mariomastrodicasa) [inline src/java/knet/src/main/java/org/mases/knet/clients/consumer/KNetConsumer.java:31]: Why?

File: src/java/knet/src/main/java/org/mases/knet/clients/consumer/KNetConsumerCallback.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ *  Refer to LICENSE for more information.
+ */
+
+package org.mases.knet.clients.consumer;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.common.annotation.InterfaceStability;
+import org.mases.jcobridge.*;
+
+@InterfaceStability.Evolving
```
Comment (mariomastrodicasa) [inline src/java/knet/src/main/java/org/mases/knet/clients/consumer/KNetConsumerCallback.java:25]: Why?

File: src/net/KNet/ClientSide/BridgedClasses/Clients/Consumer/KNetConsumer.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    {
+        readonly KNetConsumerCallback<K, V> obj;
+        internal Message(KNetConsumerCallback<K, V> obj)
+        {
+            this.obj = obj;
+        }
+
+        //public string Topic => data.TypedEventData;
+
+        //public int Partition => data.GetAt<int>(0);
```
Comment (mariomastrodicasa) [inline src/net/KNet/ClientSide/BridgedClasses/Clients/Consumer/KNetConsumer.cs:39]: Why?

Comment (masesdevelopers) [reply src/net/KNet/ClientSide/BridgedClasses/Clients/Consumer/KNetConsumer.cs:39]: Same of https://github.com/masesgroup/KNet/pull/55#discussion_r861829072. We leaved the code of a test made.

**Pull request merged**